sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -251,7 +251,10 @@ abstract class CastBase extends UnaryExpression with TimeZoneAwareExpression wit
 
   def dataType: DataType
 
-  override def toString: String = s"cast($child as ${dataType.simpleString})"
+  override def toString: String = {
+    val ansi = if (ansiEnabled) "ansi_" else ""
```

Comment:
I think it should be "ansiCast" or "ANSICast" instead of "ansi_cast"

Reply:
This follows the existing SQL function naming convention, e.g. `FromUTCTimestamp`

Reply:
But the string of `newInstance` is `newInstance($cls)` ..

Reply:
`NewInstance` is not really a SQL function/operator...

Reply:
I see. It is the first expression the overrides `toString` with 2 words in IDE...haha